examples/api-samples/package.json

```diff
@@ -7,8 +7,9 @@
     "@theia/core": "1.16.0",
     "@theia/filesystem": "1.16.0",
     "@theia/output": "1.16.0",
-    "@theia/vsx-registry": "1.16.0",
-    "@theia/workspace": "1.16.0"
+    "@theia/workspace": "1.16.0",
+    "@theia/navigator": "1.16.0",
```

Comment:
minor: add `@theia/navigator` in alphabetical order, when we perform a release `lerna` will change the order anyways.